Cargo.toml

```diff
@@ -67,6 +67,7 @@ members = [
     "examples/counter",
     "examples/custom_widget",
     "examples/download_progress",
+    "examples/winit",
```

Comment:
I think we should keep the examples as relatively standalone, full applications.

We probably don't need a new example just to showcase a new window setting. We can extend other examples instead! 😄